task4/src/main/scala/CsvTask.scala

```diff
@@ -0,0 +1,122 @@
+
```

Comment:
No package name specified
The good style is to place all the code into some package 
For example, by.itechart.stlab